library/std/intrinsic.qs

```diff
@@ -258,6 +258,17 @@ namespace Microsoft.Quantum.Intrinsic {
     /// Performs a joint measurement of one or more qubits in the
     /// specified Pauli bases.
     ///
+    /// # Description
+    /// The probability of getting `Zero` is
+    /// $\langle \psi | \frac{I + P_0 \otimes \ldots \otimes P_{N-1}}{2} | \psi \rangle$
```

Comment:
Can we use `\bra` and `\ket` rather than pipes and `\langle` and `\rangle`? We already changed the katas to be consistent here.

Reply:
Was trying to remember what we were supposed to use and failed. Fixed now.